README.md

```diff
@@ -1 +1,3 @@
 Mac Admins Open Source exists to provide the code signing certificates which are now required due to increased security requirements in macOS, to open source projects which are used in SMB, enterprise and education environments across the globe.
+
+You can find us in the #macadmins-opensource channel on the [Mac Admins slack](https://www.macadmins.org/) or on the <a rel="me" href="https://hachyderm.io/@macadminsopensource">Hachyderm</a> Mastodon server.
```

Comment:
```suggestion
You can find us in the #macadmins-opensource channel on the [Mac Admins slack](https://www.macadmins.org/) or <a rel="me" href="https://hachyderm.io/@macadminsopensource">@macadminsopensource@hachyderm.io</a> on Mastodon.
```

Reply:
This is part of the verification process:

https://community.hachyderm.io/docs/mastodon/user/verification/#how-to-verify